api/main.py

```diff
@@ -79,7 +79,7 @@ def employee_delete(id: int):
     if emp is False:
         raise HTTPException(status_code=404)
     else:
-        return True
+        raise HTTPException(status_code=200)
```

Comment:
Fastapi returns success by default, no point in raising you can remove else branch.
Best to use 204 (No Content) if no response body is sent.
`@app.delete("/employees/{id}", tags=["Employee"], status_code=204)`